doc/nrf/links.txt

```diff
@@ -197,6 +197,8 @@
 
 .. _`commit #d55314`: https://github.com/nrfconnect/sdk-nrf/commit/d553142f4f37bc324da9869e4837cdd9105e25ab
 
+.. _`Zigbee R22`: https://github.com/nrfconnect/ncs-zigbee-r22
+.. _`Zigbee R23`: https://github.com/nrfconnect/ncs-zigbee
```

Comment:
¨There could be external Zigbee links that are now no longer required with this removal. Maybe we could remove those links from the link.txt file.

Reply:
Yes, there are, but I was planning to do that in a separate PR as a part of clean-up activities. This PR should go in as soon as possible